fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -269,6 +313,13 @@ public void parseArgs(String[] args) {
                     throw new IllegalArgumentException("missing value for --bucket");
                 }
                 break;
+            case "--scan-local-dir":
```

Comment:
needs documentation.  probably in the README

Reply:
done